cmd/veneur/main.go

```diff
@@ -71,6 +71,16 @@ func main() {
 		}
 	}()
 
+	// Accept connections on the TCP port if enabled
```

Comment:
Sure. I had to rearrange the code that starts the listener to make it work with shutting down (used in the unit tests).